render.yaml

```diff
@@ -0,0 +1,35 @@
+services:
+  # Main python web service
+  - type: web
+    name: how2meet
+    runtime: python
+    repo: https://github.com/BenMemberg/how2meet.git
+    region: ohio
+    plan: free
+    branch: main
+    rootDir: how2meet
+    buildCommand: poetry install
+    startCommand: poetry run python app.py
+    envVars:
+      - fromGroup: how2meet-envvars
+    autoDeploy: true # deploy on push to main branch (testing on IAC branch)
```

Comment:
```suggestion
```
Would prefer auto-deploy be off since we only need to deploy when testing infrastructure stuff